### ISSUE TITLE
repeating same song

### ISSUE THREAD

**Issue by rghoppy** [Bleuzen/SpotRec#24]
I have tried both the original and updated version, and the recording just loops around one or two songs. 
By the way, have you looked at Audio Recorder as when the source is set to spotify it does what you are trying to achieve.

**Comment by yschroeder** [comment]
I am seeing the same issue.

BTW: Audio Recorder tends to chop off a few seconds from songs. But in theory it does the same, yes.

**Comment by jocke42** [comment]
Can confirm. While having spotrec running the playlist does not advance but jumps back to the last title after playing 1-2 seconds of the new song

Using version 0.15.1 and spotify-client 1.2.13.661 on Fedora Linux 37

**Comment by FediWarneck** [comment]
I think it's about pipewire. With pulseaudio it runs like a brave ;-)

**Comment by kaedwen** [comment]
Yes can confirm, running plain pulseaudio (no pipewire) does work :-)

**Comment by sykozwo** [comment]
I have the same problem and found a solution for me.

`_playback_time_before_seeking_to_beginning = 15.0`

I change this value from 5.0 to 15.0.
When changing tracks it normally plays 5 seconds and then uses the previous function to start from the beginning.
It seems that 5 sec. is not enough in some situations so the spot. client jumps to the last played song.
